component/client/settings/ClientSetting.tsx

```diff
@@ -104,21 +105,12 @@ class ClientSetting extends React.Component<any, IClientSettingState> {
 
   public reloadSettings() {
     this.setState({
-      dirty: false
-    });
-    const temp = this.state;
-    for (const key in temp) {
-      if (temp.hasOwnProperty(key) && key !== "dirty") {
-        storage.get(key).then((value) => {
-          const data: any = {};
-          data[key] = {
-            original: value,
-            updated: value,
-          };
-          this.setState(data);
-        });
+      dirty: false,
+      username: {
+        original: "",
```

Comment:
In the past, it reloads the value stored, so that the text field shows the current username and probably other settings. The UI is not updated based on the settings in storage.
I see this issue when I test it.